### ISSUE TITLE
chromium kali linux Pyppeteer error: Browser closed unexpectedly

### ISSUE THREAD

**Issue by azamet90** [pyppeteer/pyppeteer#152]
old issue but still exist...

```    raise BrowserError('Browser closed unexpectedly:\n')
pyppeteer.errors.BrowserError: Browser closed unexpectedly:
Pyppeteer error: Browser closed unexpectedly:
```

and couldnt understand how to fix it from old topics here... im using python 3.8.4... does pyppeteer needs a update? do i need to do somethings? or is there a workaround for this?
![Ekran görüntüsü_2020-07-24_13-53-38](https://user-images.githubusercontent.com/44042178/88364365-31699880-cdb5-11ea-833f-02638924d792.png)

**Comment by Mattwmaster58** [comment]
> does pyppeteer needs a update

Yes, it's being worked on. May or may not be related to this issue

**Comment by Mattwmaster58** [comment]
[This comment will likely give you insight as to why the process is being terminated](https://github.com/pyppeteer/pyppeteer/issues/108#issuecomment-642034439)

**Comment by azamet90** [comment]
> [This comment will likely give you insight as to why the process is being terminated](https://github.com/pyppeteer/pyppeteer/issues/108#issuecomment-642034439)

your link was very useful sir thanks!
--no-sandbox fixed my problem

**Comment by YasharZand** [comment]
If you run as root then you need the --no-sandbox config, better to run as any user rather than root for security reasons.
